src/components/ux-topbar-item/README.md

```diff
@@ -0,0 +1 @@
+> ux-topbar-item Documentation
```

Comment:
Check out the other components - what I do is take Fv5's intro text and then link to the Foundation doc page.